com.github.falkTX.Carla.json

```
@@ -0,0 +1,133 @@
+{
+  "app-id": "com.github.falkTX.Carla",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "carla",
+  "rename-desktop-file": "carla.desktop",
+  "rename-icon": "carla",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    /* MIDI */
+    "--device=all",
+    /* For pipewire */
+    "--filesystem=xdg-run/pipewire-0",
+    /* Allow loading, saving files from the home directory (portals don’t work yet) */
+    "--filesystem=home",
+    "--env=TMPDIR=/var/tmp",
+    /* On SuSE it seems to be necessary for sound to work */
+    "--env=ALSA_CONFIG_PATH=",
+    "--env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2",
```

```suggestion
    "--env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2:/app/lib/lv2",
```

You need that one too (documentation needs update).
Also TIL that `$HOME` isn't expanded. :-(

---

> You need that one too (documentation needs update). Also TIL that `$HOME` isn't expanded. :-(

Maybe Flatpak should bind mount homedir to `/var/home` like done with the xdg-dirs to /var/{config,data,cache}.

---

I filed https://github.com/flatpak/flatpak/issues/4596